gh-actions/release-images/action.yaml

```diff
@@ -14,15 +14,11 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - shell: bash
-      env:
-        IMAGES_ARGS: --nocache ${{ inputs.image_args }}
```

Comment:
Is this not needed anymore with buildx?

Reply:
We still need it; I’d restored it but hadn’t pushed the update yet.